.circleci/config.yml

```diff
@@ -130,7 +130,7 @@ commands:
             - cargo
       - restore_cache:
           keys:
-            - target-folder-{{ checksum "Cargo.lock" }}
+            - target-folder-v1-{{ checksum "Cargo.lock" }}
```

Comment:
Can you explain this change please?

Reply:
Yes, I keep getting an error during the build that is strange and only appears in the CI. I wonder if it's due to the change around the proc macro. I am renewing the target folder cache to start from fresh to see if it helps.